lib/tapioca/generator.rb

```diff
@@ -113,11 +145,42 @@ def compiler
     sig { void }
     def require_gem_file
       say("Requiring all gems to prepare for compiling... ")
-      loader.load_bundle(config.prerequire, config.postrequire)
+      begin
+        loader.load_bundle(config.prerequire, config.postrequire)
+      rescue LoadError => e
+        explain_failed_require(config.postrequire, e)
+        exit(1)
+      end
       say(" Done", :green)
       puts
     end
 
+    sig { params(file: String, error: LoadError).void }
+    def explain_failed_require(file, error)
+      say_error("\n\nLoadError: #{error}", :bold, :red)
+      say_error("\nTapioca could not load all the files required by your application.", :yellow)
```

Comment:
I wonder if we should be explicitly saying that these are "gem requires"? "All the files required by your application" sounds a bit vague.

Reply:
I went with `Tapioca could not load all the gems required by your application.`